### PR TITLE
<Halide.h> -> "Halide.h"

### PR DESCRIPTION
**Pull request by steven-johnson**
And the same for HalideRuntime.h… angle-bracket includes are supposed
to be reserved for system includes; in most build systems this is
fudged, but for some it’s hard to work around.

### CODE REVIEW

File: apps/HelloAndroid/jni/halide.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include <Halide.h>
+#include "Halide.h"
```
Comment (abadams) [inline apps/HelloAndroid/jni/halide.cpp]: If Halide is in /usr/include/Halide.h does this all still work?

File: apps/HelloAndroid/jni/native.cpp [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #include <stdlib.h>
 
 #include "halide_generated.h"
-#include <HalideRuntime.h>
+#include "HalideRuntime.h"
 #include <HalideRuntimeOpenCL.h>
```
Comment (abadams) [inline apps/HelloAndroid/jni/native.cpp]: Looks like HalideRuntimeOpenCL.h and similar was left out of this change.

File: apps/HelloAndroid/jni/halide.cpp (hunk 1 above)
Comment (steven-johnson) [inline apps/HelloAndroid/jni/halide.cpp]: ...I have no idea. Is that a use case we are likely to encounter?

File: apps/HelloAndroid/jni/native.cpp (hunk 2 above)
Comment (steven-johnson) [inline apps/HelloAndroid/jni/native.cpp]: oops, forgot about those :-(

File: apps/HelloAndroid/jni/halide.cpp (hunk 1 above)
Comment (abadams) [inline apps/HelloAndroid/jni/halide.cpp]: That's kinda the end-game (apt-get install libhalide-dev) I could imagine someone downloading a prebuilt, installing it in the usual place, and then copy-pasting code from the tests or building the tutorials.

Anyway, it looks like it works (including "stdio.h" works, at least)

**Comment by abadams** [comment]
Testing it on windows...

**Comment by abadams** [comment]
Works fine on Windows.

**Pull request merged**